src/index.ts

```diff
@@ -299,49 +305,52 @@ const createTab = async ({
-    //     return stackItem.raw;
-    //   const url = new URL(fileName);
-    //   const localFileName = path.join(process.cwd(), url.pathname);
-    //   const transformResult = await server.transformRequest(
-    //     url.pathname + url.search,
-    //   );
-    //   const map = typeof transformResult === 'object' && transformResult?.map;
-    //   if (!map) return stackItem.raw;
-
-    //   const { SourceMapConsumer } = await import('source-map');
-    //   const consumer = await new SourceMapConsumer(map as any);
-    //   const sourceLocation = consumer.originalPositionFor({ line, column });
-    //   consumer.destroy();
-    //   if (sourceLocation.line === null || sourceLocation.column === null)
-    //     return stackItem.raw;
-
-    //   const inlineCode = url.searchParams.get('inline-code');
-    //   if (inlineCode) {
-    //     const fileSrc = await fs.readFile(localFileName, 'utf8');
-    //     const inlineStartIdx = fileSrc.indexOf(inlineCode);
-    //     if (inlineStartIdx === -1) return stackItem.raw;
-    //     const linesTillInlineCode = (
-    //       fileSrc.slice(0, inlineStartIdx).match(/\n/g) || []
-    //     ).length;
-    //     column = sourceLocation.column + 1;
-    //     line = sourceLocation.line + linesTillInlineCode;
-    //   } else {
-    //     column = sourceLocation.column + 1;
-    //     line = sourceLocation.line;
-    //   }
-
-    //   fileName = localFileName;
-    //   return `    at ${fileName}:${line}:${column}`;
-    // });
+    let { message, stack } = res;
+    if (message.includes(`Failed to fetch dynamically imported module: ${url}`))
+      message =
+        'Failed to load runJS code (most likely due to a transpilation error)';
+
+    const parsedStack = parseStackTrace(stack);
+    let isFirst = true;
+    const modifiedStack = parsedStack.map(async (stackItem) => {
+      if (stackItem.raw.startsWith(stack.slice(0, stack.indexOf('\n'))))
+        return null;
+      if (!stackItem.fileName) return stackItem.raw;
+      const fileName = stackItem.fileName;
+      const line = stackItem.line;
+      const column = stackItem.column;
+      if (!fileName.startsWith(`http://localhost:${port}`))
+        return stackItem.raw;
+      const url = new URL(fileName);
+      const id = `.${url.pathname}`;
+      const transformResult = requestCache.get(id);
+      const map = typeof transformResult === 'object' && transformResult.map;
+      if (!map) return stackItem.raw;
+      const { SourceMapConsumer } = await import('source-map');
+      const consumer = await new SourceMapConsumer(map as any);
+      const sourceLocation = consumer.originalPositionFor({ line, column });
+      consumer.destroy();
+      if (sourceLocation.line === null || sourceLocation.column === null)
+        return stackItem.raw;
+      const mappedColumn = sourceLocation.column + 1;
+      const mappedLine = sourceLocation.line;
+      // The check for i === 0 is because Jest fails to recognize
```

Comment:
@calebeby I'm not sure I understand this comment. Can you elaborate? 🤔

Reply:
Oops I started writing this comment but then I wrote the comment on line 339- which is meant to replace this comment. I'll delete this comment